WordPressVIPMinimum/Sniffs/Filters/RestrictedFilterSniff.php

```diff
@@ -55,18 +57,59 @@ public function process( File $phpcsFile, $stackPtr ) {
 		}
 
 		$filterNamePtr = $phpcsFile->findNext(
-			array_merge( Tokens::$emptyTokens, array( T_OPEN_PARENTHESIS ) ), // types.
-			$stackPtr + 1, // start.
-			null, // end.
-			true, // exclude.
-			null, // value.
-			true // local.
+			array_merge( Tokens::$emptyTokens, array( T_OPEN_PARENTHESIS ) ),
+			$stackPtr + 1,
+			null,
+			true,
+			null,
+			true
+		);
+
+		$endHookPtr = $phpcsFile->findNext( 
+			T_COMMA,
+			$filterNamePtr,
+			null,
+			false,
+			null, 
+			true
 		);
 
-		$filterName = str_replace( array( "'", '"' ), '', $tokens[ $filterNamePtr ]['content'] );
+		$filterName = $this->transformString( $tokens[ $filterNamePtr ]['content'] );
+
+		// if concatenation is found, build $filterName.
+		$concatPtr = $phpcsFile->findNext(
+			T_STRING_CONCAT,
+			$filterNamePtr,
+			$endHookPtr,
+			false,
+			null, 
+			true
+		);
+		$concatenation = $concatPtr ? true: false; 
+		if ( $concatenation ) { 
+			for ( $i = $filterNamePtr + 1; $i < $endHookPtr; $i++ ) {
+				if ( T_CONSTANT_ENCAPSED_STRING === $tokens[ $i ]['code'] ) {
+					$filterName .= $this->transformString( $tokens[ $i ]['content'] );
+				}
+			}
+		}
 
 		if ( isset( $this->restrictedFilters[ $filterName ] ) && 'upload_mimes' === $filterName ) {
 			$phpcsFile->addWarning( 'Please ensure that the mimes being filtered do not include insecure types (e.g. SVG). Manual inspection required.', $stackPtr, 'UploadMimesRestrictedFilter' );
 		}
 	}
+
+	/**
+	 * Transform string.
+	 *
+	 * @param string $string The name of the filter.
+	 *
+	 * @return string
+	 */
+	private function transformString( $string ) {
+		$string = str_replace( array( "'", '"' ), '', $string ); // remove quotes (double and single).
```

Comment:
Capital needed for the start of comments, per `WordPress-Docs`.

Code base is [PHP 5.6+](https://github.com/Automattic/VIP-Coding-Standards/blob/a1e3be751a7edd8678965a504a786130db7ef293/composer.json#L13), so short array syntax can be used.